python/cudf/cudf/tests/test_list.py

```diff
@@ -297,8 +297,8 @@ def test_get_nulls():
         ([[1, 2, 3], [], [3, 4, 5]], 6, [False, False, False],),
         ([[1.0, 2.0, 3.0], None, []], 2.0, [True, None, False],),
         ([[None, "b", "c"], [], ["b", "e", "f"]], "b", [True, False, True],),
-        ([[None, 2, 3], None, []], 1, [None, None, False]),
-        ([[None, "b", "c"], [], ["b", "e", "f"]], "d", [None, False, False],),
+        ([[None, 2, 3], None, []], 1, [False, None, False]),
```

Comment:
This seems like it could affect someones downstream business logic. Should this be considered `breaking` ?

Reply:
Absosmurfly. Thanks, I neglected to update the label.

Reply:
Just updated. Meant to do so earlier as part of [this](https://github.com/rapidsai/cudf/pull/9510#issuecomment-954284749) conversation.